WooCommerce/Classes/POS/Presentation/TotalsView.swift

```diff
@@ -13,7 +13,8 @@ struct TotalsView: View {
 
     // The source of truth for whether totals _are_ showing; separate from whether they
     // _should be_ showing, so that we can animate the change.
-    @State private var isShowingTotalsFields: Bool = false
+    // Default true so totals fields would be included in the view hiearchy on first render and animate with TotalsView
```

Comment:
Nice, this perhaps explains the smother transition to totals we're seeing, where placeholders don't intersect with the cart during the animation/

Reply:
Likely. The totals fields didn't appear in TotalsView at the beginning of the Checkout transition because it wasn't in the view hierarchy when the animation began. I haven't understood the exact root cause before.

However, the totals fields are only hidden once the payment begins processing, therefore they can be safely included in the hierarchy when we open the view.